cmd/cli/cmd/page_test.go

```diff
@@ -87,36 +84,28 @@ func testCase4_5(t *testing.T, tc *testCmd) {
 	t.Log(r)
 }
 
-//testCase4_6 Delete a key in a key page
+//testCase4_6 update a key in a key page
 func testCase4_6(t *testing.T, tc *testCmd) {
 	t.Helper()
 
-	t.Log("Awaiting key page update fix for api validation error, skipping... ")
-	return
+	//replace key3 with key 4
+	commandLine := fmt.Sprintf("page key update acc://RedWagon/page1 red2 0 3 red2 red5")
+	r, err := tc.executeTx(t, commandLine)
+	require.NoError(t, err)
 
-	//uncomment after fix key page remove
-	////remove red4
-	//commandLine := fmt.Sprintf("page key remove acc://RedWagon/page2 red3 red4")
-	//r, err := tc.executeTx(t, commandLine)
-	//require.NoError(t, err)
-	//
-	//t.Log(r)
+	t.Log(r)
 }
 
-//testCase4_7 update a key in a key page
+//testCase4_7 Delete a key in a key page
 func testCase4_7(t *testing.T, tc *testCmd) {
 	t.Helper()
 
-	t.Log("Awaiting key page update fix for api validation error, skipping... ")
-	return
+	//remove red4
+	commandLine := fmt.Sprintf("page key remove acc://RedWagon/page1 red5 0 4 red5")
+	r, err := tc.executeTx(t, commandLine)
+	require.NoError(t, err)
 
-	//uncomment after key page update fix.
-	////replace key3 with key 4
-	//commandLine := fmt.Sprintf("page key update acc://RedWagon/page2 red3 red3 red4")
-	//r, err := tc.executeTx(t, commandLine)
-	//require.NoError(t, err)
-	//
-	//t.Log(r)
+	t.Log(r)
 }
```

Comment:
These test case numbers are tied to the [test matrix](https://netorg725054.sharepoint.com/:x:/r/sites/DeFiDevs/_layouts/15/Doc.aspx?sourcedoc=%7BA6FBB9EA-0C9D-47B5-8D37-83942130BB2E%7D&file=Test%20Matrix%20(1).xlsx&nav=MTBfe0VFOEYyQ0Y1LUE5NDItNEMyNS04ODI4LTUzRUI4NTRGRjJEQX1fezk0OTAwMDM4LUQ5REUtNEMxNC04QTVGLUE3QjQzMDNDODFDRH0&action=default&mobileredirect=true&cid=0e38da64-e46e-4e34-96df-be72b0af8bb5):

- 4.6: Delete Key Page
- 4.7: Update Key

`testCase4_6` must delete a key and `testCase4_7` must update a key. If necessary, the order the tests are executed in can be changed in the `init` function in this file, though there needs to be a comment explaining why.